README.md

```diff
@@ -189,17 +185,29 @@ plan myapp::patch (
 
 #### Parameters
 
-##### `targets`
+##### `nodes`
```

Comment:
Nit: targets seems more accurate than nodes? I don't feel strongly, and assume you've thought more about it. Doesn't block merge, just wondering why it changed.

Reply:
If a plan accepts `nodes` then you can use `nodes` on the CLI. So this is a pattern we’ve established for plans.